utilities/transactions/write_prepared_txn_db.h

```diff
@@ -835,8 +847,10 @@ class WritePreparedCommitEntryPreReleaseCallback : public PreReleaseCallback {
   // Either because it is commit without prepare or it has a
   // CommitTimeWriteBatch
   bool includes_data_;
-  // Should the callback also publishes the commit seq number
-  bool publish_seq_;
+  // Auxiliary batch (if there is any)
+  SequenceNumber aux_seq_;
```

Comment:
I think it'd be useful to explain the difference between aux_batch_cnt_ and data_batch_cnt_. From what I understand this only matters for two queues?